### DISCUSSION TITLE
Realtime Blog App using React+Pocketbase 💖

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#754] [Show and tell]
### I have created this awesome looking Blog Application using PocketBase and React, please have a look:
[https://github.com/rajesh6161/pbRealtimeBlog](https://github.com/rajesh6161/pbRealtimeBlog)

#### Features:
- Authentication 🔒 (login and registration basic)
- List/View 🪟 
- Create 🪄 
- Update 🧑🏻‍💻 
- Responsive

#### Some screenshots of the application:

- Home Page

![s1](https://user-images.githubusercontent.com/40054161/194911558-98b85cd6-3534-4ba0-ae49-40fb930cba64.jpg)

- Login Page

![Screenshot 2022-10-10 at 21-18-16 Realtime Blog](https://user-images.githubusercontent.com/40054161/194911374-690defb2-7ee9-46ad-ac06-b56b67c2c3fb.png)

- Single Post Page

![s2](https://user-images.githubusercontent.com/40054161/194912872-3a4f72fe-0cbd-4d39-956f-a9f006aed403.jpg)

- Create Post Page

![s3](https://user-images.githubusercontent.com/40054161/194912937-40466c86-baa9-4298-86b5-df6082f854e3.jpg)

**Comment by rajesh6161** [comment]
@ganigeorgiev

**Comment by benallfree** [comment]
This is amazing, thank you! I am the maintainer of [awesome-pocketbase](https://github.com/benallfree/awesome-pocketbase) and just added your project to the Showcases section 👍

**Reply by rajesh6161** [reply]
Thanks @benallfree I am trying to make it live and will share the link sooner.

**Comment by pragmatech** [comment]
Hi @rajesh6161 , @benallfree ,
I've been delighted to find PB and thanks to all the efforts that are being shared! Just now, initially hooking this implementation onto a PB I run on a little GCP server; the auth worked, but my post didn't fly. I'll have to go visit with the todo tutorial you kindly posted. (like is my version v0.7.6 compatible and the myriad of other potentials)
Suffice to say that the potential data sovereignty, and the minimalism WITH websockets is exciting. Mostly wanted to say hey, and thanks. Perhaps i need to step up a offer some more tangible PR. aloha.
@rajesh6161 your comment "am trying to make it live" has me thinking that, for troubleshooting etc.; knowing in more detail about your trials in making it live (sounds like Frankn) ... or simply 'it don't work yet' would be educational! thanks
edit #3 - checking your git comments(duh), you do indeed offer up the scoops re:progress etc. ...the short story is realtime isn't realtiming yes? thanks

**Reply by rajesh6161** [reply]
@pragmatech thanks for your kind words! indeed pocketbase is an excellent project. 
Can you please tell me where you are having issues? you can raise an issue so that I can get good idea.

**Comment by dimplast** [comment]
@rajesh6161 great project. Pocketbase is awesome. But how you deploy pocketbase in Vercel? Have you any instructions about this?

**Reply by ganigeorgiev** [reply]
@dimplast You can host your frontend in Vercel but as far as I know Vercel doesn't offer persistent storage, so you'll need to host PocketBase somewhere else. The project in this discussion seems to use @benallfree's [pockethost.io](https://pockethost.io/). Alternatively, you can also check [fly.io](https://github.com/pocketbase/pocketbase/discussions/537).

_You can find generic deployment instructions in https://pocketbase.io/docs/going-to-production/._

**Reply by rajesh6161** [reply]
@dimplast I am glad you liked it, no doubt pocketbase is best in class. As @ganigeorgiev  mentioned I have used pockethost.io for hosting pocketbase and Vercel for the frontend.
I hope it helps you.

**Reply by benallfree** [reply]
@rajesh6161 frontend hosting coming soon to https://pockethost.io

**Comment by dimplast** [comment]
@rajesh6161 Thank you for your instructions. I used pocketbase.io for hosting pocketbase and Vercel for the frontend for my little (register-login-logout) sveltekit project. For now everything are ok. 

@benallfree pockethost.io is awesome. The frontend hosting will be perfect.

@ganigeorgiev pocketbase.io is an excellent solution. The deployment (on server) for me is very hard because i have not experiense with docker and i work in windows environment. (any advice will be wellcome)

**Comment by kumard3** [comment]
hi @rajesh6161 where do you host your pocketbase instance?

**Reply by benallfree** [reply]
I'm not sure where he is hosting it, but pockethost.io is an option for that kind of app.

**Reply by kumard3** [reply]
Ok i know i am being crazy but is it possible to somehow host pocket base on vercel or netlify or cloudflare ?

**Reply by rajesh6161** [reply]
hey @kumard3  I have hosted on @benallfree 's pockethost.io and its awesome!

**Reply by kumard3** [reply]
Ya that is really cool, but i am not able to find any pricing page ?

**Reply by benallfree** [reply]
@kumard3 You can jump over to PocketHost.io discussion area, but short answer: free til v1 which is probably 6 months away.
